app/src/main/java/io/apicurio/registry/events/http/HttpEventSink.java

```diff
@@ -73,33 +69,35 @@ public void handle(Message<Buffer> message) {
     private void sendEventHttp(String type, HttpSinkConfiguration httpSink, Buffer data) {
         try {
             log.debug("Sending event to sink " + httpSink.getName());
-            getHttpClient()
-                .request(new RequestOptions()
-                            .setMethod(HttpMethod.POST)
-                            .setURI(httpSink.getEndpoint())
-                            .putHeader("ce-id", UUID.randomUUID().toString())
-                            .putHeader("ce-specversion", "1.0")
-                            .putHeader("ce-source", "apicurio-registry")
-                            .putHeader("ce-type", type)
-                            .putHeader("content-type", MediaType.APPLICATION_JSON),
-                            ar -> {
-                                if (ar.succeeded()) {
-                                    ar.result()
-                                        .exceptionHandler(ex -> log.error("Error sending event to " + httpSink.getEndpoint(), ex))
-                                        .end(data);
-                                } else {
-                                    log.error("Error sending event to " + httpSink.getEndpoint(), ar.cause());
-                                }
-                            });
+
+            final HttpRequest eventRequest = HttpRequest.newBuilder()
+                    .uri(URI.create(httpSink.getEndpoint()))
+                    .version(HttpClient.Version.HTTP_1_1)
+                    .header("ce-id", UUID.randomUUID().toString())
+                    .header("ce-specversion", "1.0")
+                    .header("ce-source", "apicurio-registry")
+                    .header("ce-type", type)
+                    .header("content-type", MediaType.APPLICATION_JSON)
+                    .POST(HttpRequest.BodyPublishers.ofByteArray(data.getBytes()))
+                    .build();
+
+            final HttpResponse<String> eventResponse = getHttpClient()
+                    .send(eventRequest, HttpResponse.BodyHandlers.ofString());
+
+            if (eventResponse.statusCode() != 200) {
+                log.error("Error sending http event: {}", eventResponse.body());
```

Comment:
I don't have the whole context, so I might be saying the wrong thing. But I find surprising seeing a log call at `error` level without an explicit `fail()` or throw of exception.

Reply:
You're actually right, this should be a warning, not an error. Long day...